backend/grading/grading_test.go

```diff
@@ -0,0 +1,84 @@
+package grading
+
+import (
+	"strings"
+	"strconv"
+	"testing"
+	"math"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestParseOutput(t *testing.T) {
```

Comment:
There's already a function named this, so if you could merge the two that'd be sweet.

Reply:
Ah, got it. I must have glazed over parser_test.go during the first push.